hub/osaka/constraints/instruction-handling/call/precompiles/modexp/common/__06__pricing_row.lisp

```diff
@@ -23,23 +23,29 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
 
+
+(defun    (precompile-processing---MODEXP---call-OOB-on-pricing-row)       (shift    misc/OOB_FLAG    precompile-processing---MODEXP---misc-row-offset---pricing))
+
+
+
 (defconstraint    precompile-processing---MODEXP---pricing-analysis---setting-misc-module-flags
                   (:guard    (precompile-processing---MODEXP---standard-precondition))
                   ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
                   (eq!    (weighted-MISC-flag-sum    precompile-processing---MODEXP---misc-row-offset---pricing)
-                          MISC_WEIGHT_OOB
+                          (*  MISC_WEIGHT_OOB  (precompile-processing---MODEXP---all-byte-sizes-are-in-bounds))
```

Comment:
Conditional OOB calls: we invoke `MODEXP_PRICING` iff `all-byte-sizes-are-in-bounds = <true>`